docs/contributing.md

```diff
@@ -14,7 +16,64 @@ Nano has a relatively small community of developers and only a handful of develo
+
+Pitch: Nano is a strongly undervalued cryptocurrency. Both as a medium of exchange and store of value, it is fundamentally the best there is in the space. As a medium of exchange, Nano being instant, feeless and scalable means that it is hard to imagine something better. As a store of value, Nano's zero inflation, eco-friendliness and game theory leading to ever more decentralization over time make it potentially the best store of value. However, many cryptocurrencies have higher market caps than Nano. If one analyses any other cryptocurrency that claims to be a store of value or medium of exchange, they'll find that Nano is fundamentally better. Because of this, it stands to reason that as the market gets more mature and rational, Nano's recognition and price will increase.
+
+##### Open source developers
+
+##### Exchanges
+
+Why: A currency becomes more useful and valuable when more people own it. The easier it is for people to exchange into Nano, the better. At the same time, the easier it is to sell Nano, the more likely it is that people will use accept Nano payments, knowing they can exchange into fiat easily if they so desire. More exchange listings also leads to a more decentralized network, as the Nano supply becomes more spread out.
+
+Pitch: Nano is interesting both from a direct business and a marketing perspective. In terms of direct business, Nano has a sizeable and growing trading volume. It has clear usecases in being used for payments and remittances. Many of those receiving Nano want to exchange into fiat or other cryptocurrencies. Additionally, many new Nano holders are new to crypto as a whole, and are looking for an exchange to buy Nano on. Nano also has a large and vocal community. Any exchange listing is broadly shared on all sorts of social mediums (Twitter, Reddit) and is one of the best forms of free marketing available.
+
+##### Entrepeneurs
+
+##### Payment Processors
+
+Why: A currency becomes more useful and valuable when there are more places to spend it. As Nano is first and foremost a currency, more merchant acceptance is key. Many merchants use payment processors, hence this being a very important avenue for Nano growth.
+
+Pitch: Nano is a cryptocurrency that is literally made for payments. It's instant and feeless, exposing you to minimal costs and minimal volatility risk. Because of its usability as currency, Nano has built up a large community interested in spending Nano. Adding Nano as a payment option would lead to increased volume, and also a lot of free marketing from Nano enthusiasts recommending your payment processor service to merchants looking to integrate crypto payments.
```

Comment:
remove or replace "literally". 

I think it would be good to mention that Nano's scalability and feeless nature has the potential to generate more transaction volume than alternatives, which will generally mean more revenue for payment processors.